### PR TITLE
Last remaining metadata files

### PR DESCRIPTION
**Pull request by ChrisFrenchPDX**
PR for the`Cloud & Portal`, `Edit data`, `Route & Direction`, and `Display Information` category metadata files.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Display information/Add graphics with renderer/README.metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Add graphics with renderer/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Add graphics with symbols/README.metadata.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Add graphics with symbols/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Control graphic draw order/README.metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Control graphic draw order/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Display grid/README.metadata.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Display grid/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/README.metadata.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Identify graphics/README.metadata.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Identify graphics/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Picture marker symbols/README.metadata.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Picture marker symbols/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/README.metadata.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Show callout/README.metadata.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Show callout/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Display information/Show legend/README.metadata.json [hunk 10]
```diff
@@ -0,0 +1,33 @@
+{
+    "category": "Display  Information",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Display information/Show legend/README.metadata.json]: Should this just be 1 space between these 2 words

File: arcgis-ios-sdk-samples/Route & Directions/Find route/README.metadata.json [hunk 11]
```diff
@@ -0,0 +1,39 @@
+{
+    "category": "Route and Directions",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Route & Directions/Find route/README.metadata.json]: Does this need to be Route & Navigation to match the folder structure in the sample repo? ... or is it OK to be Route and Navigation (which is more readable).

File: arcgis-ios-sdk-samples/Route & Directions/Find service area interactive/README.metadata.json [hunk 12]
```diff
@@ -0,0 +1,33 @@
+{
+    "category": "Route and Directions",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Route & Directions/Find service area interactive/README.metadata.json]: Does this need to be Route & Navigation to match the folder structure in the sample repo? ... or is it OK to be Route and Navigation (which is more readable).

File: arcgis-ios-sdk-samples/Route & Directions/Offline routing/README.metadata.json [hunk 13]
```diff
@@ -0,0 +1,38 @@
+{
+    "category": "Route and Directions",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Route & Directions/Offline routing/README.metadata.json]: Does this need to be Route & Navigation to match the folder structure in the sample repo? ... or is it OK to be Route and Navigation (which is more readable).

File: arcgis-ios-sdk-samples/Route & Directions/Route around barriers/README.metadata.json [hunk 14]
```diff
@@ -0,0 +1,40 @@
+{
+    "category": "Route and Directions",
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Route & Directions/Route around barriers/README.metadata.json]: Does this need to be Route & Navigation to match the folder structure in the sample repo? ... or is it OK to be Route and Navigation (which is more readable).

File: arcgis-ios-sdk-samples/Route & Directions/Find route/README.metadata.json (hunk 11 above)
Comment (ChrisFrenchPDX) [reply arcgis-ios-sdk-samples/Route & Directions/Find route/README.metadata.json]: It should be fine: what matters the most is that `category` property values share identical strings if they have the same category.

**Pull request merged**